server/chat-commands/info.ts

```diff
@@ -2530,6 +2530,63 @@ export const commands: ChatCommands = {
 	},
 	denyshowhelp: [`/denyshow [user] - Denies the media display request of [user]. Requires: % @ # &`],
 
+	randquote(target, room, user) {
+		room = this.requireRoom();
+		if (!room.settings.quotes?.length) return this.errorReply(`This room has no quotes.`);
+		this.runBroadcast();
+		const [quote] = Utils.shuffle(room.settings.quotes);
```

Comment:
Wait, this is wrong. `Utils.shuffle` is a mutating shuffle which should not be applied to the quotes list. If there's no `Utils.random` you should be doing:

```suggestion
		const quote = room.settings.quotes[Math.floor(Math.random() * room.settings.quotes.length)];
```